### PR TITLE
OKTA-661610 Update rate limit docs

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updated the best practices with existing and new info; added better cross-doc links; added link to support page on requesting temporary rate limit increases.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-661610 ](https://oktainc.atlassian.net/browse/OKTA-661610 )

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/rl-best-practices/index.md [hunk 1]
```diff
@@ -183,9 +192,17 @@ The error condition resolves itself as soon as there is another concurrent threa
 }
 ```
 
-### Request exceptions
+## Review your API limit parameter
+
+In some cases, you can avoid hitting rate limits by implementing the maximum value of the `limit` query parameter for an individual API endpoint. Increasing this value to the maximum reduces the number of calls for a given operation, which can keep you under the endpoint's rate limit. By default, Okta uses a default `limit` if one is not set.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-best-practices/index.md:197]:
```suggestion
In some cases, you can avoid hitting rate limits by implementing the maximum value of the `limit` query parameter for an individual API endpoint. Increasing this value to the maximum reduces the number of calls for a given operation, which can keep you under the endpoint's rate limit. By default, Okta uses a default `limit` if one isn't set.
```

File: packages/@okta/vuepress-site/docs/reference/rl-best-practices/index.md [hunk 2]
```diff
@@ -183,9 +192,17 @@ The error condition resolves itself as soon as there is another concurrent threa
...
+
+For example, the endpoint `/api/v1/apps/${applicationId}/users` returns, by default, 50 results. You can increase this limit to 500, reducing the number of calls.
+
+See the [Core Okta API](/docs/reference/core-okta-api/) for details on each endpoint's default and maximum values for the `limit` parameter.
+
+## Request rate limit exceptions
 
-You can request a temporary rate limit increase. For example, if you are importing a large number of users and groups, you may need a temporary rate limit increase.
+The size and structure of rate limits can be adjusted. You can request a temporary rate limit increase. For example, if you are importing a large number of users and groups, you may need a temporary rate limit increase. See [How to Request a Temporary Rate Limit Increase](https://support.okta.com/help/s/article/How-can-we-request-to-have-the-rate-limit-for-our-org-temporarily-increased?language=en_US).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-best-practices/index.md:205]:
```suggestion
You can adjust the size and structure of rate limits by requesting a temporary rate limit increase. For example, if you are importing a large number of users and groups, you may need a temporary rate limit increase. See [How to Request a Temporary Rate Limit Increase](https://support.okta.com/help/s/article/How-can-we-request-to-have-the-rate-limit-for-our-org-temporarily-increased?language=en_US).
```

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md [hunk 3]
```diff
@@ -89,6 +89,8 @@ Each API is a link that navigates you to the [dashboard view of API rate limits
 
 You can perform a search in your table list by making an entry in the Search box. You can also perform a "lookup" with the Search. For example, you're calling Okta with an example `requestURI` and you want to know the rate limit that is applied to calls that are made against that `requestURI`.
 
+API endpoints in the dashboard that end with an asterisk (for example, `/api/v1/users/*`) generally refer to calls that use that base schema and count towards the overall limit, unless the endpoint is specifically called out in the dashboard. It's also important to note the HTTP operation involved with each call, as the same call with different operations can count towards different limits.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:92]:
```suggestion
API endpoints in the dashboard that end with an asterisk (for example, `/api/v1/users/*`) refer to calls that use that base schema and count towards the overall limit, unless the endpoint is specifically called out in the dashboard. It's also important to note the HTTP operation involved with each call, as the same call with different operations can count towards different limits.
```

File: packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md [hunk 4]
```diff
@@ -11,10 +11,10 @@ This page provides the API rate limits for authentication and end-user activitie
...
 > * [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/) apply to a variety of endpoints across different APIs for customers that purchased this add-on.
-> * You can expand Okta rate limits upon request. To learn how, see [Request exceptions](/docs/reference/rl-best-practices/#request-exceptions) and [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/).
+> * You can expand Okta rate limits upon request. To learn how, see [Request exceptions](/docs/reference/rl-best-practices/#request-rate-limit-exceptions) and [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/).
 >
 
-See the following list of **per-minute** limits. If an endpoint is not in this list, you can review it using the Admin Console, in the rate limit dashboard's APIs table. See [APIs table](/docs/reference/rl-dashboard/#apis-table).
+See the following list of per-minute limits. If an endpoint is not in this list, you can review it using the Admin Console, in the rate limit dashboard's APIs table. See [APIs table](/docs/reference/rl-dashboard/#apis-table).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-enduser/index.md:17]:
```suggestion
See the following list of per-minute limits. If an endpoint isn't in this list, you can review it using the Admin Console, in the rate limit dashboard's APIs table. See [APIs table](/docs/reference/rl-dashboard/#apis-table).
```

File: packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md [hunk 5]
```diff
@@ -14,10 +14,10 @@ This page provides the API rate limits for management activities, which is part
...
 > * [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/) apply to a variety of endpoints across different APIs for customers that purchased this add-on.
 > * In addition to the rate limit per API, Okta implements limits on concurrent requests, Okta-generated email messages, end user requests, and home page endpoints. These limits are described on the [Additional limits](/docs/reference/rl-additional-limits/) page.
-> * You can expand Okta rate limits upon request. To learn how, see [Request exceptions](/docs/reference/rl-best-practices/#request-exceptions) and [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/).
+> * You can expand Okta rate limits upon request. To learn how, see [Request exceptions](/docs/reference/rl-best-practices/#request-rate-limit-exceptions) and [DynamicScale rate limits](/docs/reference/rl-dynamic-scale/).
 >
 
-We enforce the following **per-minute** limits:
+See the following list of per-minute limits. If an endpoint is not in this list, you can review it using the Admin Console, in the rate limit dashboard's APIs table. See [APIs table](/docs/reference/rl-dashboard/#apis-table).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/rl-global-mgmt/index.md:20]:
```suggestion
See the following list of per-minute limits. If an endpoint isn't in this list, you can review it using the Admin Console, in the rate limit dashboard's APIs table. See [APIs table](/docs/reference/rl-dashboard/#apis-table).
```

**Review by susanharper-okta — APPROVED** [review]
couple nits - approving to not hold you up.

File: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md (hunk 3 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md:92]: @susanharper-okta I added "generally" here to be purposefully vague. Support can configure things that can make this statement incorrect.

**Pull request merged**